crudd/src/main.rs

```diff
@@ -124,8 +124,11 @@ async fn cmd_read<T: BlockIO>(opt: &Opt, crucible: Arc<T>) -> Result<()> {
     // interested in seeing how the pipeline length affects performance. I
     // don't think it'll make a difference beyond 2, (one reading from
     // crucible, one writing to output), but we'll see!
+    //
```

Comment:
Maybe @faithanalog has some thoughts here on how this comment should be updated?

Reply:
I opened https://github.com/oxidecomputer/crucible/issues/459 to track this.